src/transformers/trainer.py

```diff
@@ -3225,6 +3225,8 @@ def evaluation_loop(
                 if self.preprocess_logits_for_metrics is not None:
                     logits = self.preprocess_logits_for_metrics(logits, labels)
                 logits = self.accelerator.gather_for_metrics((logits))
+                if args.eval_logits_to_cpu:
+                    logits = logits.cpu()
```

Comment:
this would fail if the logits are a tuple of tensors which can be the case as per below line in prediction step:

https://github.com/huggingface/transformers/blob/e66984f9952641e176521a7ef3faf2d5fa3c608b/src/transformers/trainer.py#L3440

Reply:
Agreed. Can we bring in `to_device` from `accelerate` instead maybe? (I didn't see a similar util in transformers, but safe to say we can use it here)